### DISCUSSION TITLE
Would graphql-java organization willing to accept project donation?

### DISCUSSION THREAD

**Discussion by dugenkui03** [graphql-java/graphql-java#2571] [General]
Hi,

I'd like to know would graphql-java origination accept project donation. 

[GraphQl Java](https://github.com/graphql-calculator/graphql-calculator) is a lightweight tool which could be used to alter execution behavior of **graphql query**.  such as:

```graphql
query basicMapValue($userIds:[Int]){
    userInfoList(userIds:$userIds)
    {
        id
        age
        firstName
        UPPER_CASE_FIRST_NAME: firstName @map(mapper:"firstName")
        lastName
        fullName: lastName @map(mapper: "firstName + lastName")

    }
}

query filterUserByAge($userId:[Int]){
    userInfoList(userIds: $userId)
    @filter(predicate: "age>=18")
    {
        userId
        age
        firstName
        lastName
    }
}

query passFetchedValueToAnotherFieldArgumentMap($itemIds:[Int]){
    commodity{
        itemList(itemIds: $itemIds){
            # save sellerId as List<Long> with unique name "sellerIdList"
            sellerId @fetchSource(name: "sellerIdList")
            name
            saleAmount
            salePrice
        }
    }

    consumer{
        userInfoList(userIds: 1)
        # transform the argument of "userInfoList" named "userIds" according to expression "sellerIdList" and expression argument, 
        # which mean replace userIds value by source named "sellerIdList"
        @argumentTransform(argumentName: "userIds", 
            expression: "sellerIdList", 
            dependencySources: ["sellerIdList"]
        ){
            userId
            name
            age
        }
    }
}
```

This is a brief introduction: https://github.com/graphql-calculator/graphql-calculator/blob/main/README_EN.md. Compared to [Schema Directives](https://www.graphql-tools.com/docs/schema-directives), query directives will be more flexible, because your can describe different execution behavior for one FIELD_DEFINITION.

In the company I work, Lots of  scenario use it to process the query.I'd like to make more people to use and complete it. Would graphql-java organization willing to accept the ownership of graphql-calculator.

Best wishes.

**Comment by bbakerman** [comment]
It is unlikely we would accept - each library we have requires maintenance and we are arguably not doing a good enough job now maintaining them

But thank you for the offer

**Comment by dugenkui03** [comment]
Thanks for your response. I am pleasure to try again when you are ready to accept project donation, and I will surely keep on maintaining this project.
